lib/win-rebase.js

```diff
@@ -0,0 +1,7 @@
+var re = new RegExp(/(.*((node\.exe($| ))|(node($| ))|(iojs($| ))|(iojs\.exe($| )))) ?(.*)$/)
+
+module.exports = function (path, rebase) {
+  var m = path.match(re)
```

Comment:
@rmg this is just what fell out of my fingers, no good reason -- I find I flip flop a lot on how I build up regexes.